client/src/components/css/QA.css

```diff
@@ -67,6 +66,7 @@ td {
   
 .qa-popup {
   position: fixed;
+  z-index: 5;
```

Comment:
What does this do?

Reply:
haha I don't remember what I doing with that, may be I test something and I forgot to delete it :)